src/components/LinkCard/LinkCard.types.ts

```diff
@@ -0,0 +1,5 @@
+import type { LinkType } from '../../modules/Dashboard/Dashboard.types'
```

Comment:
Import should be shorthanded to `"../../modules/Dashboard"`

Add .types file to `index.ts`